test/e2e/ginkgo/ginkgo_suite_test.go

```diff
@@ -39,20 +39,30 @@ func TestGinkgo(t *testing.T) {
 }
 
 var (
-	ctx           context.Context
-	coreCluster   *kubernetes.Clientset
-	wds           *kubernetes.Clientset
-	ksWds         *ksClient.Clientset
-	wec1          *kubernetes.Clientset
-	wec2          *kubernetes.Clientset
-	its           *ocmWorkClient.Clientset
-	releasedFlag  bool
-	skipSetupFlag bool
+	ctx                context.Context
+	coreCluster        *kubernetes.Clientset
+	wds                *kubernetes.Clientset
+	ksWds              *ksClient.Clientset
+	wec1               *kubernetes.Clientset
+	wec2               *kubernetes.Clientset
+	its                *ocmWorkClient.Clientset
+	releasedFlag       bool
+	skipSetupFlag      bool
+	hostClusterCtxFlag string
+	wds1CtxFlag        string
+	its1CtxFlag        string
+	wec1CtxFlag        string
+	wec2CtxFlag        string
 )
 
 func init() {
 	flag.BoolVar(&releasedFlag, "released", false, "released controls whether we use a release image")
 	flag.BoolVar(&skipSetupFlag, "skip-setup", false, "skip kubestellar cleanup and setup")
+	flag.StringVar(&hostClusterCtxFlag, "hostCluster-ctx", "kind-kubeflex", "context for kubeflex hosting cluster")
+	flag.StringVar(&wds1CtxFlag, "wds1-ctx", "wds1", "context for KS wds1 space")
+	flag.StringVar(&its1CtxFlag, "its1-ctx", "its1", "context for KS its1 space")
+	flag.StringVar(&wec1CtxFlag, "wec1-ctx", "cluster1", "context for wec1 cluster")
+	flag.StringVar(&wec2CtxFlag, "wec2-ctx", "cluster2", "context for wec2 cluster")
```

Comment:
You might want to spell out "context" in the flag name. That is what we do in other cases (https://github.com/kubestellar/kubestellar/blob/v0.22.0/options/client-options.go#L44), and I have seen other projects do that too (e.g., `--kube-context` in `helm create`). Not a blocker, I will leave it up to you.

/lgtm

Reply:
@MikeSpreitzer, thanks! I fixed that to spell out "context" in the flag name.